lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -396,7 +397,11 @@ func (handler *ContainersHandlersImpl) GetContainerLogsHandler(params containers
 		tail = int(*params.Taillines)
 	}
 
-	reader, err := container.LogReader(context.Background(), tail, follow)
+	if *params.Since > 0 {
+		since = *params.Since
+	}
+
```

Comment:
If `since` isn't specified, it actually comes in from the docker client as `-62135596800` :)